src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resource/PrisonResourceTest.kt

```diff
@@ -0,0 +1,63 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.resource
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.mock.mockito.MockBean
+import uk.gov.justice.digital.hmpps.welcometoprison.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.welcometoprison.service.PrisonService
+
+@Suppress("ClassName")
+class PrisonResourceTest : IntegrationTestBase() {
+  @MockBean
+  private lateinit var prisonService: PrisonService
```

Comment:
As you're dealing with images here rather than the standard JSON responses it might be nice to stub the endpoint with Wiremock and return a real image etc.